src/Pact/Types/KeySet.hs

```diff
@@ -192,22 +192,18 @@ instance Arbitrary KeySetName where
 instance Pretty KeySetName where
   pretty ksn = "'" <> pretty (asString ksn)
 
-keysetNameParser
-  :: TokenParsing m
-  => Monad m
-  => m KeySetName
+keysetNameParser :: Parser KeySetName
 keysetNameParser =
   qualifiedKeysetNameParser <|> woNs
   where
-    woNs = (`KeySetName` Nothing) <$> (ident style <* eof)
+    -- legacy support for keyset names must support both
+    -- whitespace spaces as well as names like "SB <2>".
+    woNs = (`KeySetName` Nothing) <$> (takeText <* eof)
 
-qualifiedKeysetNameParser
-  :: TokenParsing m
-  => Monad m
-  => m KeySetName
+qualifiedKeysetNameParser :: Parser KeySetName
 qualifiedKeysetNameParser = do
   ns <- NamespaceName <$> ident style
-  kn <- dot *> ident style <* eof
+  kn <- dot *> takeText <* eof
```

Comment:
why are we using `takeText` here? qualified keyset name parsing _should_ be enforcing whatever we want to enforce. This one can be `ident style` (as long as this supports hex all charas for hashes, which I think it does)

Reply:
Okay, so the rationale here is that people, if they have keyset names like `SV <2>` whill want to migrate them from the root space into a namespace like `ns.SV <2>`. Since we already have to support the old style, it makes sense for consistency's sake that we keep it and only constrain the namespace. WyT?

Reply:
Talked with stuart, and it looks like we can constrain this.